### PR TITLE
Completed exercise 1

### PR DESCRIPTION
**Pull request by evo777**
Completed test function.

### CODE REVIEW

File: test-add/test.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const add = require('./add.js');
+
+const test = (a, b, c) => {
+  if (add(a, b) === c) {
+    return 'Correct';
+  }
+  return 'Incorrect';
+};
+
+console.log(test(1, 2, 3));
```
Comment (songz) [inline test-add/test.js]: wouldn't it be more efficient to move console.log into the function so you don't have to repeat it over and over?

File: test-add/test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const add = require('./add.js');
+
+const test = (a, b, c) => {
```
Comment (songz) [inline test-add/test.js]: could a,b,c have better names? num1, num2, sum?

**Comment by evo777** [comment]
Added in your suggestions.

File: test-add/add.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+function add(num1, num2) {
```
Comment (pkdotson) [inline test-add/add.js]: changes to ES6 notation! :)

**Review by pkdotson — APPROVED** [review]

**Pull request merged**